logstash/class-logger.php

```diff
@@ -306,13 +306,14 @@ protected static function parse_params( array $params ) : array {
 		];
 
 		if ( ! isset( $params['file'] ) && ! isset( $params['line'] ) ) {
+			// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_debug_backtrace
 			$backtrace = debug_backtrace( DEBUG_BACKTRACE_IGNORE_ARGS, 1 );
 
-			if ( isset( $backtrace[0] ) && isset( $backtrace[0]['file'] ) ) {
+			if ( isset( $backtrace[0]['file'] ) ) {
 				$default_params['file'] = $backtrace[0]['file'];
 			}
 
-			if ( isset( $backtrace[0] ) && isset( $backtrace[0]['line'] ) ) {
+			if ( isset( $backtrace[0]['line'] ) ) {
```

Comment:
**Possible bug:** this method is protected; `$backtrace[0]` refers to the caller. The caller will be `Logger::log2logstash()`. Shouldn't we check the first frame, not the zeroth one?

Reply:
Pinging @hanifn for your input on this based on your PR at https://github.com/Automattic/vip-go-mu-plugins/pull/1530/files.